### ISSUE TITLE
Add MV3 build to CI and release build

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#7171]
Context
---
- The remaining blockers have been merged: RJSF and template sandbox
- So we should be able to create MV3 builds of the extension for testing
- In future work, we will create an MV3 build for submission to a CWS store beta listing, but that's blocked on adding app support for multiple chrome extension ids

Implementation Sketch
---
- Update webpack configuration as necessary
- Update the CI build workflow to include an MV3 build
- Update the release build to include an MV3. Use the same manifest id as the mv3 build for now. (In a follow-up ticket, we'll create build(s) for the CWS beta listing

Definition of Done
---
- CI builds an MV3 build (to ensure the build works)
- Release builds include an MV3 build with the same manifest key so we can perform internal regression testing/dogfooding
- Remove `GOOGLE_*` env variables

Discussion
---
- Can we close out?: https://github.com/pixiebrix/pixiebrix-extension/issues/287
- We might want to create a completely different release workflow for GitHub actions? 
- We might need to turn of Rollbar source maps for the MV3 build? https://github.com/pixiebrix/pixiebrix-extension/blob/4aff813ef450c232bf5f012d1bd71650ed905136/.github/workflows/release.yaml#L10-L10
- While you're in the GitHub actions, you can drop the `GOOGLE_*` environment variables

Related Code
---
- See the MV flag: https://github.com/pixiebrix/pixiebrix-extension/blob/4aff813ef450c232bf5f012d1bd71650ed905136/webpack.config.js#L425-L425

**Comment by fregante** [comment]
I can work on this if you'd like

**Comment by fregante** [comment]
> * Can we close out?: [Identify gaps preventing migration to Chrome Manifest V3 #287](https://github.com/pixiebrix/pixiebrix-extension/issues/287)

I think to do that we need to actually run the extension in MV3 and see what's still broken (which I assume it's still a bit and likely more than before, namely data that is not in storage.session)

**Comment by twschiller** [comment]
@fregante I think @grahamlangford is creating the build today

> I think to do that we need to actually run the extension in MV3 and see what's still broken (which I assume it's still a bit and likely more than before, namely data that is not in storage.session)

I started a PR with a lint rule here: https://github.com/pixiebrix/pixiebrix-extension/pull/7180

Looks like there are at least a couple places the could be migrated. Could you take a look?

**Comment by grahamlangford** [comment]
>I can work on this if you'd like

>@fregante I think @grahamlangford is creating the build today

If you've got the extra bandwidth, that would be a big help. I've got a lot of meetings today, and I'll be out next week. I'd like to get this in sooner rather than later.

**Comment by twschiller** [comment]
@fregante reopening because the build is not working correctly:

See screenshot the `build-production-3` is still manifest v2:

<img width="827" alt="image" src="https://github.com/pixiebrix/pixiebrix-extension/assets/1879821/3ec0b8cd-0eba-461c-ad6b-72d14f19080b">
